rubix-spi/src/test/java/com/qubole/rubix/spi/TestCacheUtil.java

```diff
@@ -153,8 +153,8 @@ public void testGetLocalPath()
 
     createCacheDirectoriesForTest(conf);
 
-    String localPath = CacheUtil.getLocalPath(remotePath, conf);
-    assertEquals(localPath, cacheTestDirPrefix + "0" + "/fcache/" + localRelPath, "Paths not equal!");
+    String localPath = CacheUtil.getLocalPath(remotePath, conf, 0);
+    assertEquals(localPath, cacheTestDirPrefix + "0" + "/fcache/" + localRelPath + "_g0", "Paths not equal!");
```

Comment:
Write a method here that returns expectedPath and use for all tests in this file. Make that method use UNKNOWN_GENERATION_NUMBER based prefix instead of hardcoding `_g1`